src/server/config.rs

```diff
@@ -252,6 +261,8 @@ impl Default for Config {
             // Go tikv client uses 4 as well.
             forward_max_connections_per_address: 4,
             simplify_metrics: false,
+            metrics_compact_level: MetricsCompactLevel::NoCompact,
+            metrics_level: MetricsLevel::All,
```

Comment:
I'm a bit of confused by these two `level`, is it possible to unify these two parameters?

Reply:
In my original design, I also combine these two configs into one and only provide 4 levels. @kevin-xianliu
 thinks it's better and more flexible to use two configs because they are two different dimensions.

Reply:
In my mind, there are two dimensions to reduce metrics traffic . One is adjusting the accuracy of metrics, the other is adjusting the total number of active metrics.  If we use one parameter to control above two dimensions, it might be hard to extend in the future?

Reply:
Although, I do think we can use clearer names instead of  metrics_compact_level and metrics_level.

Reply:
better to use a clearer name.